### DISCUSSION TITLE
Content for the community page of the website

### DISCUSSION THREAD

**Discussion by kjaymiller** [BlackPythonDevs/blackpythondevs#16] [General]
Right now it just reads "Sample Content for the page" we should put something there. Perhaps we can have it talk about all the areas people can connect with us.

Discord - I think we can open the doors a little more now. use invite link - https://discord.gg/XUc3tFqCT3
X - https://twitter.com/blackpythondevs

I would also be open to discussing a Mastodon/Instragram/and LinkedIn Group

I have to admit that I wouldn't be active on the IG one but I could keep an eye on the LinkedIn Community and remind folks that most of the discussion is happening in the Discord.

**Comment by kjaymiller** [comment]
cc @VeldaKiara @oleksis

**Comment by VeldaKiara** [comment]
I would recommend try to first build engagement on the Discord, LinkedIn and Twitter then build on the others as we grow

**Reply by kjaymiller** [reply]
I definitely think we can build engagment on LinkedIn if we have content. We can also cross promote to help spark conversation in the Discord as well... I don't have much hopes for Twitter

**Reply by VeldaKiara** [reply]
Twitter can be for news or announcements that we have

**Reply by oleksis** [reply]
With Twitter/X we share the news about Python Ecosystem and Community like Us, Events, Conferences, Live streams...

**Comment by VeldaKiara** [comment]
For the community page we can have information about the plans we have like our Friday meetups and also highlight how people can get involved by contributing to our projects or something. it would also help if we have dicussions on there too. Like if you learn something, you can post it on the community and have people engage on it. You can also start something like, what does removing GIL  in Python mean for the users etc. You can explain what you know it means and opens it up for folks to join

**Comment by oleksis** [comment]
> For the community page we can have information about the plans we have like our Friday meetups and also highlight how people can get involved by contributing to our projects or something. 

I agree 👍🏾. We need the content and remember this need be translated 

> Like if you learn something, you can post it on the community and have people engage on it. 

On Discord we start this idea with channels named "What you learned this week?" . We can keep similar ideas, discuss in the community and create the **Content**

**Reply by kjaymiller** [reply]
The what you learned/did this week was hopefully a way to encourage others to get engaged
